be/src/exprs/string_functions.cpp

```diff
@@ -1049,6 +1049,14 @@ Status StringFunctions::pad_prepare(FunctionContext* context, FunctionContext::F
     state->fill_is_const = false;
     context->set_function_state(FunctionContext::FRAGMENT_LOCAL, state);
 
+    if (context->get_num_args() == 2) {
```

Comment:
it is not required to modify this function impl, just handling default param in FE is just convenient. 
please modify AstBuilder.visitSimpleFunctionCall to add default param for lpad/rpad/pad function.

Reply:
Get it.